TaskmanAPI/Controllers/ProjTasksController.cs

```diff
@@ -65,4 +65,26 @@ public async Task<ActionResult<ProjTask>> AssignUsers(int id, string username)
         var userId = await _userService.GetUserIdByUsername(username);
         return Ok(await _projTasksService.AddUser(id, userId));
     }
+
+    // api/Tasks/{id}/remove_user/{username}
+    [HttpDelete("{id}/remove_user/{username}")]
+    public async Task<ActionResult<ProjTask>> RemoveUsers(int id, string username)
+    {
+        var userId = await _userService.GetUserIdByUsername(username);
+        return Ok(await _projTasksService.RemoveUser(id, userId));
+    }
+
+    // api/Tasks/{id}/change_status/{status}
+    [HttpPut("{id}/change_status/{status}")]
+    public async Task<ActionResult<ProjTask>> ChangeStatus(int id, string status)
```

Comment:
Changing status does not result in strings declared in Enum, it changes into something else (WaitingForActivation etc)

Reply:
should be fixed in latest commit